src/components/App.js

```diff
@@ -26,9 +26,10 @@ import mapConst             from "../constants/Map"
 class Main extends Component {
   
   render(){
+    
     const { dispatch, search, view, server, map, form, url, user, t } = this.props;
     const { entries, ratings } = server;
-
+    console.log(map)
```

Comment:
@tragladit May you explain this string a little bit?

Reply:
@fr0zen this string not needed. Forgot to delete ))

Reply:
@tragladit please remove this line and commit, changes must appear here)